### ISSUE TITLE
Missing AudioKitUI.xcframework when adding AudioKit 5.0.b1 from CocoaPods 

### ISSUE THREAD

**Issue by idcuesta** [AudioKit/AudioKit#2184]
Hi there,

I am trying to get AudioKit to compile using Xcode 12 within my project, which makes use of AudioKitUI to display EZAudioPlot view.

When I install AudioKit version 5.0.b1 as a cocoapods dependency (using cocoapods version 1.9.3), no matter if I just add default library or try to add directly UI subspec, I always get same result. 

```
source 'https://github.com/AudioKit/Specs.git'
...
pod 'AudioKit', '5.0.b1'
#or
pod 'AudioKit/UI', '5.0.b1'
```

Pods project only includes `AudioKit.xcframework`, and `AudioKitUI.xcframework` is missing.
<img width="233" alt="Screenshot 2020-07-20 at 21 45 44" src="https://user-images.githubusercontent.com/2726409/87979809-e9a3f200-cad2-11ea-9094-bf5192f70600.png">

Is this expected? I guess it is not, because I can see in [Specs/AudioKit/5.0.b1/AudioKit.podspec.json](https://github.com/AudioKit/Specs/blob/master/AudioKit/5.0.b1/AudioKit.podspec.json) that there is a subspec `UI` that is supposed to return a vendored framework.
<img width="496" alt="Screenshot 2020-07-20 at 21 50 33" src="https://user-images.githubusercontent.com/2726409/87979954-25d75280-cad3-11ea-971c-7041835b7a58.png">

Any help would be appreciated 🙏

**Comment by megastep** [comment]
That is odd, indeed it should have both frameworks if you use `AudioKit/UI` - I see that the pod spec is also a little bit behind but that doesn't explain this.

**Comment by megastep** [comment]
Actually, I just tried here (with Xcode 11 though), and both get installed properly, although UI is in its own folder under Pods:
 
<img width="266" alt="image" src="https://user-images.githubusercontent.com/115222/87993681-f3603200-ca9f-11ea-9316-f09cdb266ee1.png">

**Comment by idcuesta** [comment]
Actually, for me in my setup it was working but started failing in CI. Then I cleared CocoaPods cache by doing `pod cache clean --all` and started failing locally too. Would you mind trying it?

**Comment by megastep** [comment]
Just did a `pod cache clean --all` followed by a `pod install` and I got both frameworks... Haven't checked with Xcode 12 though, but I doubt that would be the cause of it?

**Comment by idcuesta** [comment]
Thanks for checking it ... this is weird. What version of CocoaPods are you using? I am on 1.9.3

**Comment by megastep** [comment]
I'm on 1.9.3 as well...

**Comment by idcuesta** [comment]
After some checks I got it working fine after doing `pod repo update`, it looks I had some old version of AudioKit specs

Thank you for your help!

```
 ~/workspace/xxx/ [development*] pod repo update
Updating spec repo `audiokit`
  $ /usr/local/bin/git -C /Users/nacho/.cocoapods/repos/audiokit fetch origin --progress
  remote: Enumerating objects: 5, done.        
  remote: Counting objects: 100% (5/5), done.        
  remote: Compressing objects: 100% (2/2), done.        
  remote: Total 5 (delta 2), reused 5 (delta 2), pack-reused 0        
  From https://github.com/AudioKit/Specs
     f518c43..c1511bb  master     -> origin/master
  $ /usr/local/bin/git -C /Users/nacho/.cocoapods/repos/audiokit rev-parse --abbrev-ref HEAD
  master
  $ /usr/local/bin/git -C /Users/nacho/.cocoapods/repos/audiokit reset --hard origin/master
  HEAD is now at c1511bb [Fix] AudioKit (5.0.b1)
```
